fork-visualizer/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: "3.7"
+
+services:
+  visualizer:
+    image: ghcr.io/polkadot-blockchain-academy/fork-visualizer-dev:latest
```

Comment:
we will change this to `ghcr.io/polkadot-blockchain-academy/fork-visualizer:latest` once this PR is merged.